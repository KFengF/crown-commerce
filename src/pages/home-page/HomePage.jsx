import React from 'react';
import Directory from './components/directory/Directory';
import './HomePage.scss';

const HomePage = () => (
    <div className="home-page" >
        <Directory />
    </div>
);

export default HomePage;