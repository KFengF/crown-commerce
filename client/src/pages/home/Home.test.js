import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './Home.page';

it('should render Home page', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
});