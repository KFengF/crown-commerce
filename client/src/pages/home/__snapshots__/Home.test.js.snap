// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home page 1`] = `ShallowWrapper {}`;
